app/assets/stylesheets/components/_icon.scss

```diff
@@ -12,3 +12,22 @@
     background-image: url(image-path('ico-print.svg'));
   }
 }
+
+.ico-absolute {
+  background-repeat: no-repeat;
+  background-size: $h4;
```

Comment:
I think it's fine as is, if we find that we need to use something similar again in the future maybe the mixin route would be the way to go at that point.  What I meant about the `h4` was just that this is more of a heading style at this point, only used with `h4`'s so I think it might make more sense to have it in the `_typography.scss` vs `_icon.scss`.